Deleting_Content.md

```diff
@@ -0,0 +1,50 @@
+# Things Are Sometimes Better Unread
+
+In any large digitial community with a diverse group of humans, communication can break down. Whether it's unintential miscommunication or intentional boundary pushing, Administrators are occasionally requested to delete a message someone wrote in the public channels.
+
+# In Brief
+
+* Admins may delete messages that violate the Code of Conduct.
+* In non-trolling cases, the authors of those messages will be notified and may be given a chance to modify the message themselves.
+
+# Don't Be a Troll
+
+A troll is a member who starts flame wars or intentionally upsets other members by posting inflammatory and digressive, extraneous, or off-topic messages with the intent of provoking members into displaying emotional responses and normalizing tangential discussion, either for the troll's amusement or a specific gain.
+
+The determination of whether a message is trolling is often a subjective assessment by administrators, but there is a specific situation where The MacAdmins Slack has a zero-tolerance policy where:
+
+1. A message which is perceived and/or reported as trolling,
```

Comment:
This first example is slightly incongruous with a zero-tolerance policy. Specifically "perceived as" should maybe not result in no-warning expulsion

Reply:
I've hopefully clarified that all three points need to be met for a no-warning expulsion. 

"there is a specific situation where The MacAdmins Slack has a zero-tolerance policy where all of the following are met:"